src/runtime/test/slot-composer-tests.ts

```diff
@@ -121,10 +121,10 @@ recipe
     const verifySlot = (fullName) => {
       const slot = slotComposer.consumers.find(s => fullName === s.consumeConn.getQualifiedName());
       assert.equal(MockSlotDomConsumer, slot.constructor);
-      assert.isTrue(Boolean(slotComposer._contexts.find(context => context === slot.slotContext)));
+      assert.isTrue(Boolean(slotComposer.contexts.find(context => context === slot.slotContext)));
     };
     const verifyHostedSlot = (fullName) => {
-      const slot = slotComposer.consumers.find(s => fullName === s.consumeConn.getQualifiedName());
+      const slot = slotComposer.consumers.find(s => fullName === s.consumeConn.getQualifiedName()) as HostedSlotConsumer;
```

Comment:
will `as HostedSlotComposer` conversion fail, if the object isn't a HostedSlotComposer? if so, you could remove the assert on the next line.

Reply:
We still need the assert.  Only a proper instanceof check would do the right thing.